src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/factory/StandardPlayerRequestResolverFactory.java

```diff
@@ -51,11 +51,11 @@ public StandardPlayerRequestResolver getNewInstance(
                                                  @NotNull final Object... context)
       throws IllegalArgumentException
     {
-        ILocation location;
+        final ILocation location;
         try {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)